scripts/apps/desks/services/DesksFactory.ts

```diff
@@ -133,6 +134,7 @@ export function DesksFactory($q, api, preferencesService, userList, notify,
                     _.each(result, (user) => {
                         self.userLookup[user._id] = user;
                     });
+                    UserActions.initUsers(self.users._items);
```

Comment:
Could you fetch users to redux store on application start and instead of `userList.getAll()` above, get them from the redux store? It'd be good not to rely on angular services for new code.

Reply:
What about this one? ^

Reply:
I would rather do it in a separate PR if we go for this redux approach, it won't change the master desk code